pkg/arkd-wallet-nbxplorer/core/application/wallet/key_manager.go

```diff
@@ -0,0 +1,169 @@
+package wallet
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/btcsuite/btcd/btcutil/hdkeychain"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+)
+
+type keyManager struct {
+	// m/84'/0'/0'
+	mainAccount *hdkeychain.ExtendedKey
+	// m/86'/0'/1'
+	connectorAccount *hdkeychain.ExtendedKey
+	// m/86'/0'/0'
+	arkSignerAccount *hdkeychain.ExtendedKey
```

Comment:
this account at some point won't be used anymore, let's use 1 for it so we won't have a gap later